custom/lib_portal/models/product_template.py

```diff
@@ -0,0 +1,19 @@
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _name = "product.template"
+    _inherit = 'product.template'
+    
+    authorName = fields.Text('Author')
+    bookDescription = fields.Text('Book Description')
+    publcationDate = fields.Date('Publication Date')
+    bookLanguage = fields.Text(string='Book Language')
```

Comment:
* This field should be a many2one to 'res.lang'

Reply:
Thank you.i didnt think of res.lang until now.Fixed it